### ISSUE TITLE
QueryList query.changes.subscribe is not a function

### ISSUE THREAD

**Issue by tlancina** [angular/angular#4635]
From the CHANGELOG: 
```
Before: query.onChange(() => ...);
After: query.changes.subscribe((iterable) => {});
```

This throws `query.changes.subscribe is not a function` in alpha.40: http://plnkr.co/edit/8hcr2WIjmSuwiAV0wEmN?p=preview (see container.ts)

It works with query.changes._subject.subscribe though, so I'm not sure if it's a documentation issue or a bug :)

**Comment by ericmartinezr** [comment]
@tlancina it is in fact a bug. You can use `query.changes.toRx().subscribe()` for now. [EventEmitter is being refactored](https://github.com/angular/angular/pull/4604) (returned by `changes`), so in futures releases you'll be able to use directly `query.changes.subscribe())

**Comment by tlancina** [comment]
@ericmartinezr great thanks, that feels much better than using the private property.

**Comment by todoubaba** [comment]
A duplicate of https://github.com/angular/angular/issues/4496.
For now:
```typescript
query.changes.observer({
	next: () => {
		...
	}
});
```

**Comment by tlancina** [comment]
Ah didn't see the dup, closing.
